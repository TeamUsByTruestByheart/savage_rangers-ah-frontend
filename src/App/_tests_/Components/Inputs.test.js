import React from 'react';
import { shallow } from '../../../enzyme';
import Input from '../../Components/Inputs';

const props = {
  className: '',
  inputMode: '',
  onChange: jest.fn(),
  autoComplete: '',
  required: false
};

describe('Input component', () => {
  const component = shallow(<Input {...props} />);
  it('renders the input component', () => {
    expect(component).toMatchSnapshot();
  });
});
