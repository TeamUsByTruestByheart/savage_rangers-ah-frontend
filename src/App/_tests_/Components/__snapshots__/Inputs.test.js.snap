// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component renders the input component 1`] = `ShallowWrapper {}`;
